app/presenters/alchemy/base_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module Alchemy
+  class BasePresenter
```

Comment:
It wouldn't conflict with the already existing serializers, but it doesn't use `ActiveModel::Serializer` from the `active_model_serializers` gem because of shortcomings in its implementation. Instead, it relies on AR's own built-in serialization. Is it okay to mix up both AR's serialization and `active_model_serializers` serialization in one directory?